extensions/interactions/base.py

```diff
@@ -143,6 +143,25 @@ def _stats_log_template(self):
             return '{{answer}}'
 
     @property
+    def rules(self):
```

Comment:
Thinking about it, should this be renamed rules_dict? rules suggests a domain object...